.pre-commit-config.yaml

```diff
@@ -1,52 +1,63 @@
----
+ci:
+  autofix_commit_msg: "style(pre-commit): autofix"
+  autoupdate_commit_msg: "ci(pre-commit): autoupdate"
+
 repos:
   - repo: https://github.com/Lucas-C/pre-commit-hooks
-    rev: v1.1.10
+    rev: v1.5.5
     hooks:
+      - id: forbid-crlf
+      - id: remove-crlf
+      - id: forbid-tabs
       - id: remove-tabs
 
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.0.1
+    rev: v4.6.0
     hooks:
       - id: trailing-whitespace
+        args: [--markdown-linebreak-ext=md]
       - id: check-merge-conflict
       - id: end-of-file-fixer
+      - id: mixed-line-ending
       - id: check-added-large-files
       - id: check-case-conflict
       - id: check-json
       - id: check-yaml
-        args:
-          - '--allow-multiple-documents'
+        args: [--unsafe, --allow-multiple-documents]
       - id: check-symlinks
       - id: detect-private-key
 
-  - repo: https://github.com/adrienverge/yamllint.git
-    rev: v1.26.3
+  - repo: https://github.com/adrienverge/yamllint
+    rev: v1.35.1
     hooks:
       - id: yamllint
         files: \.(yaml|yml)$
         types: [file, yaml]
         entry: yamllint --strict
+        exclude: .*.param.yaml
 
   - repo: https://github.com/igorshubovych/markdownlint-cli
-    rev: v0.29.0
+    rev: v0.40.0
     hooks:
-    - id: markdownlint
+      - id: markdownlint
+        args: [-c, .markdownlint.yaml, --fix]
 
   - repo: https://github.com/Yelp/detect-secrets
-    rev: v1.1.0
+    rev: v1.5.0
     hooks:
-    - id: detect-secrets
-      args: ['--baseline', '.secrets.baseline']
-      exclude: .*/tests/.*
+      - id: detect-secrets
+        args: [--baseline, .secrets.baseline]
```

Comment:
Usually I'd expect to see quotes in making an args list like this.  What's the guideline/reason that prompted to remove the quotes?

Reply:
> Best Practices:
> For simple argument values like file paths or numeric parameters, using args without quotes is common and considered best practice.